--- libwapcaplet-0.2.0/Makefile~	2013-04-19 20:52:56.000000000 +0300
+++ libwapcaplet-0.2.0/Makefile	2013-04-27 15:44:41.144316739 +0300
@@ -42,6 +42,7 @@
 
 # Extra installation rules
 I := /include/libwapcaplet
+lib = lib
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/libwapcaplet/libwapcaplet.h
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib/pkgconfig:lib$(COMPONENT).pc.in
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib:$(OUTPUT)
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib)/pkgconfig:lib$(COMPONENT).pc.in
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib):$(OUTPUT)
