--- libwapcaplet-0.2.2/Makefile.orig	2015-03-16 11:53:38.726130704 +0100
+++ libwapcaplet-0.2.2/Makefile	2015-03-16 11:54:43.132448452 +0100
@@ -18,7 +18,7 @@ WARNFLAGS := -Wall -W -Wundef -Wpointer-
 	-Wmissing-declarations -Wnested-externs
 # BeOS/Haiku standard library headers issue warnings
 ifneq ($(BUILD),i586-pc-haiku)
-  WARNFLAGS := $(WARNFLAGS) -Werror
+  WARNFLAGS := $(WARNFLAGS)
 endif
 CFLAGS := -D_BSD_SOURCE -I$(CURDIR)/include/ \
 	-I$(CURDIR)/src $(WARNFLAGS) $(CFLAGS)
